osbuild/testutil/net.py

```diff
@@ -25,6 +27,16 @@ def __init__(self, *args, **kwargs):  # pylint: disable=unused-argument
 from .atomic import AtomicCounter
 
 
+def print_dir(directory):
+    if os.path.isdir(directory):
```

Comment:
Sorry, what I meant was that `os.walk()` can simplify this, helper, something like:
```python
def print_dir(directory):
    for root, _, files in os.walk(directory):
        for fn in files:
            print(os.path.join(root, fn))
```